projects/js-packages/charts/package.json

```diff
@@ -66,6 +67,7 @@
 		"@visx/xychart": "^3.12.0",
 		"clsx": "2.1.1",
 		"date-fns": "^4.1.0",
+		"lodash": "^4.17.21",
```

Comment:
I only intended to import `lodash.merge` but eslint says to do this

Reply:
The Rollup config in the project only transpiles each TS file to JS file without any dependencies. And it is up to the consumer's bundler to decide whether do tree shake or bundle it all. In conclusion, I think this is all good 👍